cli/internal/login/link.go

```diff
@@ -143,34 +173,69 @@ func (c *LinkCommand) Run(args []string) int {
 		}
 	}
 	fs.EnsureDir(filepath.Join(".turbo", "config.json"))
-	fsErr := config.WriteRepoConfigFile(&config.TurborepoConfig{
+	err = config.WriteRepoConfigFile(&config.TurborepoConfig{
 		TeamId: chosenTeam.ID,
-		ApiUrl: c.Config.ApiUrl,
+		ApiUrl: l.apiURL,
 	})
-	if fsErr != nil {
-		c.logError(fmt.Errorf("could not link current directory to team/user.\n%w", fsErr))
-		return 1
+	if err != nil {
+		return fmt.Errorf("could not link current directory to team/user.\n%w", err)
 	}
 
-	if !dontModifyGitIgnore {
+	if l.modifyGitIgnore {
 		fs.EnsureDir(".gitignore")
 		_, gitIgnoreErr := exec.Command("sh", "-c", "grep -qxF '.turbo' .gitignore || echo '.turbo' >> .gitignore").CombinedOutput()
 		if err != nil {
-			c.logError(fmt.Errorf("could find or update .gitignore.\n%w", gitIgnoreErr))
-			return 1
+			return fmt.Errorf("could find or update .gitignore.\n%w", gitIgnoreErr)
 		}
 	}
 
-	c.Ui.Info("")
-	c.Ui.Info(util.Sprintf("%s${RESET} Turborepo CLI authorized for ${BOLD}%s${RESET}", ui.Rainbow(">>> Success!"), chosenTeamName))
-	c.Ui.Info("")
-	c.Ui.Info(util.Sprintf("${GREY}To disable Remote Caching, run `npx turbo unlink`${RESET}"))
-	c.Ui.Info("")
-	return 0
+	l.ui.Info("")
+	l.ui.Info(util.Sprintf("%s${RESET} Turborepo CLI authorized for ${BOLD}%s${RESET}", ui.Rainbow(">>> Success!"), chosenTeamName))
+	l.ui.Info("")
+	l.ui.Info(util.Sprintf("${GREY}To disable Remote Caching, run `npx turbo unlink`${RESET}"))
+	l.ui.Info("")
+	return nil
 }
 
 // logError logs an error and outputs it to the UI.
-func (c *LinkCommand) logError(err error) {
-	c.Config.Logger.Error("error", err)
-	c.Ui.Error(fmt.Sprintf("%s%s", ui.ERROR_PREFIX, color.RedString(" %v", err)))
+func (l *link) logError(err error) {
+	l.logger.Error("error", err)
+	l.ui.Error(fmt.Sprintf("%s%s", ui.ERROR_PREFIX, color.RedString(" %v", err)))
+}
+
+func promptSetup(location string) (bool, error) {
+	shouldSetup := true
+	err := survey.AskOne(
+		&survey.Confirm{
+			Default: true,
+			Message: util.Sprintf("Would you like to enable Remote Caching for ${CYAN}${BOLD}\"%s\"${RESET}?", location),
+		},
+		&shouldSetup, survey.WithValidator(survey.Required),
+		survey.WithIcons(func(icons *survey.IconSet) {
+			// for more information on formatting the icons, see here: https://github.com/mgutz/ansi#style-format
+			icons.Question.Format = "gray+hb"
+		}))
+	if err != nil {
+		return false, err
+	}
+	return shouldSetup, nil
+}
+
+func promptTeam(teams []string) (string, error) {
+	chosenTeamName := ""
+	err := survey.AskOne(
+		&survey.Select{
+			Message: "Which Vercel scope (and Remote Cache) do you want associate with this Turborepo? ",
+			Options: teams,
```

Comment:
`survey` supports pagination, `ApiClient` supports pagination, we should remember to go back and add the ability to request more.